security-admin/src/main/java/org/apache/ranger/rest/PublicAPIs.java

```diff
@@ -415,4 +416,38 @@ public RangerManagementConfig getManagementConfiguration(@Context HttpServletReq
 		return rangerManagementConfig;
 	}
 
+	@GET
+	@Path("/admin/ok")
+	public String checkRangerHealth (@Context HttpServletRequest request) {
+
+		if (logger.isDebugEnabled()) {
+			logger.debug("==> PublicAPIs.checkRangerHealth(): ");
+		}
+		HashMap<String, String> responseMap = new HashMap<>();
+
+		// Default Response Parameters
+		String serviceStatus = "ok";
+		String dbStatus = "ok";
+		String version = RangerVersionInfo.getVersion();
+
+		// Get Database Version
+		String dbVersion = daoMgr.getXXDataHist().getDBVersion();
+
+		if (dbVersion.equalsIgnoreCase("Not Available")) {
+			serviceStatus = "partially running";
+			dbStatus = "not running";
+		}
+		responseMap.put("ranger-admin-service", serviceStatus);
```

Comment:
Though we are not checking for response json in test, however, altiscale pattern for admin/ok response is like
{
   "service-name": true|false
   ok:true|false
   version:x.x
}
This is what we get for KMS
https://kms1.test.altiscale.com:48163/admin/ok
{
 “machashy-service”: true,
 “ok”: true,
 “version”: “1.0"
}

Reply:
this is the "altiscale" way to monitor. For ranger, it's already different because of the url prefix. But, we can make this to be more similar - better example would be KDC-REST service that has LDAP as a backend.